packages/messaging-api-messenger/src/MessengerClient.ts

```diff
@@ -521,6 +521,88 @@ export default class MessengerClient {
     return this.deleteMessengerProfile(['persistent_menu'], options);
   }
 
+  /**
+   * User Level Persistent Menu
+   *
+   * https://developers.facebook.com/docs/messenger-platform/send-messages/persistent-menu#user_level_menu
+   */
+  getUserPersistentMenu(
+    userId: string,
+    { accessToken: customAccessToken }: Types.AccessTokenOptions = {}
+  ): Promise<Types.PersistentMenu | null> {
+    return this._axios
+      .get(
+        `/me/custom_user_settings?psid=${userId}
```

Comment:
`/me/custom_user_settings?psid=${userId}&access_token=${customAccessToken || this._accessToken}` should be in single line.